src/diffusers/pipelines/flux/pipeline_flux_fill.py

```diff
@@ -838,9 +897,37 @@ def __call__(
             lora_scale=lora_scale,
         )
 
-        # 4. Prepare latent variables
+        # 4. Prepare timesteps
+        sigmas = np.linspace(1.0, 1 / num_inference_steps, num_inference_steps) if sigmas is None else sigmas
+        image_seq_len = (int(height) // self.vae_scale_factor // 2) * (int(width) // self.vae_scale_factor // 2)
+        mu = calculate_shift(
+            image_seq_len,
+            self.scheduler.config.base_image_seq_len,
+            self.scheduler.config.max_image_seq_len,
+            self.scheduler.config.base_shift,
+            self.scheduler.config.max_shift,
+        )
```

Comment:
```suggestion
        mu = calculate_shift(
            image_seq_len,
            self.scheduler.config.get("base_image_seq_len", 256),
            self.scheduler.config.get("max_image_seq_len", 4096),
            self.scheduler.config.get("base_shift", 0.5),
            self.scheduler.config.get("max_shift", 1.15),
        )
```

We need to keep this for scheduler compatibility, see https://github.com/huggingface/diffusers/pull/10384#issue-2758882751

Reply:
@Suprhimp want to update the code or should I do it and merge?

Reply:
Than can I ask for help for this part please? 😂

Reply:
We only need to hit "Commit suggestion" button

Reply:
yeah I know but I always like to give the opportunity first to the contributor (to learn) and also the commit has a different author if we do it.